### PR TITLE
fix bug mobile : search channel tab can't scroll

### PR DESCRIPTION
**Pull request by loi-huynhphuc**

### CODE REVIEW

File: apps/mobile/src/app/components/MembersSearchTab/index.tsx [hunk 1]
```diff
@@ -21,17 +21,17 @@ type MembersSearchTabProps = {
...
 		setSelectedUser(null);
 	};
 	return (
 		<View style={styles.container}>
 			{listMemberSearch?.length > 0 ? (
-				<ScrollView showsVerticalScrollIndicator={false}>
+				<ScrollView contentContainerStyle={{ paddingBottom: size.s_50 }} showsVerticalScrollIndicator={false}>
 					{
-						<Block width={'100%'} marginTop={size.s_10} borderRadius={size.s_14} backgroundColor={themeValue.secondary}>
+						<View style={{ width: '100%', marginTop: size.s_10, borderRadius: size.s_14, backgroundColor: themeValue.secondary }}>
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/MembersSearchTab/index.tsx]: style inline

File: apps/mobile/src/app/components/ThreadDetail/ActionRow/index.tsx [hunk 2]
```diff
@@ -16,7 +16,13 @@ enum EActionRow {
...
 	Threads,
 	Mute,
-	Settings
+	Settings,
+}
+
+export enum EOpenSearchChannelFrom {
+	ChannelList,
+	HeaderDefault,
+	ActionMenu,
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/ThreadDetail/ActionRow/index.tsx]: move to lib

File: apps/mobile/src/app/screens/home/homedrawer/HomeDefault.tsx [hunk 3]
```diff
@@ -29,6 +29,7 @@ import BottomKeyboardPicker from './components/BottomKeyboardPicker';
 import EmojiPicker from './components/EmojiPicker';
 import LicenseAgreement from './components/LicenseAgreement';
 import { style } from './styles';
+import { EOpenSearchChannelFrom } from '../../../components/ThreadDetail/ActionRow';
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/HomeDefault.tsx]: import from lib

File: apps/mobile/src/app/screens/home/homedrawer/ChannelList/index.tsx [hunk 4]
```diff
@@ -29,6 +29,7 @@ import { ChannelListSection } from '../components/ChannelList/ChannelListSection
 import ChannelMenu from '../components/ChannelMenu';
 import ClanMenu from '../components/ClanMenu/ClanMenu';
 import { style } from './styles';
+import { EOpenSearchChannelFrom } from '../../../../../app/components/ThreadDetail/ActionRow';
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/ChannelList/index.tsx]: import from lib

File: apps/mobile/src/app/navigation/ScreenTypes.ts [hunk 5]
```diff
@@ -3,6 +3,7 @@ import { IChannel, OptionEvent } from "@mezon/utils";
 import { CompositeScreenProps, NavigatorScreenParams } from "@react-navigation/native";
 import { NativeStackScreenProps } from "@react-navigation/native-stack";
 import { StackScreenProps } from "@react-navigation/stack";
+import { EOpenSearchChannelFrom } from "../components/ThreadDetail/ActionRow";
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/navigation/ScreenTypes.ts]: import from lib

File: apps/mobile/src/app/components/ThreadDetail/SearchMessageChannel/InputSearchMessageChannel/index.tsx [hunk 6]
```diff
@@ -41,7 +43,11 @@ const InputSearchMessageChannel = ({ onChangeText }: InputSearchMessageChannelPr
 					navigation.goBack();
 				}}
 			>
-				<AngleLeft width={20} height={20} color={Colors.textGray} />
+				{openSearchChannelFrom === EOpenSearchChannelFrom.ActionMenu ? (
+					<AngleLeft width={20} height={20} color={Colors.textGray} />
+				) : openSearchChannelFrom === EOpenSearchChannelFrom.ChannelList ? (
+					<ArrowLeftIcon width={20} height={20} color={Colors.textGray} />
+				) : null}
```
Comment (sontung-nccvn) [inline apps/mobile/src/app/components/ThreadDetail/SearchMessageChannel/InputSearchMessageChannel/index.tsx]: use ```&&``` operator or useMemo instead

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**